graph/api/src/main/java/org/jboss/windup/graph/model/ProjectModel.java

```diff
@@ -225,12 +225,22 @@
     /**
      * Returns the project model that represents the whole application. If this projectModel is the root projectModel, it will return it.
      *
+     * Note: This may be the synthetic shared-libs project in some cases.
+     *
      * @return ProjectModel representing the whole application
      */
     @JavaHandler
     ProjectModel getRootProjectModel();
 
     /**
+     * Returns all applications that this project is a part of. This could be multiple applications if this project
+     * is included multiple times.
```

Comment:
So it means something like "get parent applications"? 
`getApplications()` method sort of implies these would be child applications.

Reply:
+1. `getCointainingApplications()` ?

Reply:
I'm just adding a few notes from the discussion from this morning for reference.

getApplications() is called that as there is no notion of "child" applications in Windup. The concept of an application is that the application is the root project and thus would never have a parent (by definition).